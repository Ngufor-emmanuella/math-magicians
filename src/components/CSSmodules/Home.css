.welcome {
  color: rgb(17, 17, 73);
  padding: 2rem 2rem 0 2rem;
  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
  font-weight: 500;
  font-size: 28px;
}

.head {
  padding: 0 2rem;
  color: #000;
  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
}

.main-box {
  width: 90%;
  padding: 0 2rem;
  font-weight: 500;
  font-size: 22px;
  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
}
