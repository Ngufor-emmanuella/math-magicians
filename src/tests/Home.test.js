import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/Home';

test('should match the snapshot', () => {
  const home = renderer.create(<Home />);
  const tree = home.toJSON();
  expect(tree).toMatchSnapshot();
});
