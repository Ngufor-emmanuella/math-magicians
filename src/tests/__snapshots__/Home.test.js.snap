// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<main>
  <h2
    className="welcome"
  >
    Welcome to our page
  </h2>
  <div
    className="main-box"
  >
    <p
      className="sub-box"
    >
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Laborum maxime a ex quaerat nesciunt nostrum labore blanditiis voluptatum, repellendus sed, odio sit eveniet natus harum nisi voluptatem amet. Consequatur, consectetur. Lorem ipsum dolor sit amet consectetur adipisicing elit. Laborum maxime a ex quaerat nesciunt nostrum labore blanditiis voluptatum, repellendus sed, odio sit eveniet natus harum nisi voluptatem amet. Consequatur, consectetur.
    </p>
    <p
      className="lores"
    >
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Laborum maxime a ex quaerat nesciunt nostrum labore blanditiis voluptatum, repellendus sed, odio sit eveniet natus harum nisi voluptatem amet. Consequatur, consectetur.Lorem ipsum dolor sit amet consectetur adipisicing elit. Laborum maxime a ex quaerat nesciunt nostrum labore blanditiis voluptatum, repellendus sed, odio sit eveniet natus harum nisi voluptatem amet. Consequatur, consectetur.
    </p>
  </div>
</main>
`;
